addons/point_of_sale/models/pos_order.py

```diff
@@ -1360,7 +1360,7 @@ def write(self, values):
         return super().write(values)
 
     @api.model
-    def get_existing_lots(self, company_id, product_id):
+    def get_existing_lots(self, company_id, config_id, product_id):
```

Comment:
You can't do this stable. If you really need this information, you should put it in the context and access it via `self.env.context`.